monai/bundle/scripts.py

```diff
@@ -678,6 +679,48 @@ def run(
     workflow.finalize()
 
 
+def run_workflow(workflow: str | BundleWorkflow | None = None, args_file: str | None = None, **kwargs: Any) -> None:
+    """
+    Specify `bundle workflow` to run monai bundle components and workflows.
```

Comment:
We should elaborate a little more here on what customised workflow classes would be and how they would operate. What's required here is to create a subclass of `BundleWorkflow` then pass it in or mention it by name, which you can on the command line. We should just state the user needs to provide that subclass and be available to import, the documentation for how to do that is elsewhere.

Reply:
Sure, added doc-string in the latest commit, thanks.